prometheus_api_client/prometheus_connect.py

```diff
@@ -40,6 +40,7 @@ class PrometheusConnect:
     :param proxy: (Optional) Proxies dictionary to enable connection through proxy.
         Example: {"http_proxy": "<ip_address/hostname:port>", "https_proxy": "<ip_address/hostname:port>"}
     :param session (Optional) Custom requests.Session to enable complex HTTP configuration
+    :param timeout: (int) A timeout (in seconds) applied to all requests
```

Comment:
nitpick:  `:param timeout: (int) A timeout (in seconds) applied to all requests` => `:param timeout: (Optional) A timeout (in seconds) applied to all requests`

Reply:
```suggestion
    :param timeout: (Optional) A timeout (in seconds) applied to all requests
```

Reply:
Addressed in https://github.com/4n4nd/prometheus-api-client-python/pull/280/commits/e0dd4a623bb9659cc27a7be51abc9b05668696b1